routes/order.js

```diff
@@ -19,7 +19,7 @@ module.exports = function (express) {
       data.uuid = uuid_generator.generateUUID();
       data.timestamp = timestamp.makeTimestamp();
       // ending the response and console logging the response data
-      console.log(data);
+      res.send({healthy: true});
```

Comment:
With how the routes are set up, how do you display the new data to the user? Is it maybe because we are using a get method before using the put method?